src/components/GameView.js

```diff
@@ -47,7 +47,7 @@ export const GameView = (props) => {
     const stalemate = chess.in_stalemate();
     const threefoldRepetition = chess.in_threefold_repetition();
     const insufficientMaterial = chess.insufficient_material();
-    chess.turn() === "b" ? setPlayer("White") : setPlayer("Black");
+    chess.turn() === "b" ? setPlayer("black") : setPlayer("white");
```

Comment:
This logic is wrong. This is for determining the winner of the game. What we had was right. Don't reuse player state but make a new state for orientation.